src/nvim/popupmenu.c

```diff
@@ -1319,6 +1320,14 @@ static void pum_position_at_mouse(int min_width)
   int col = mouse_col;
   pum_win_row_offset = 0;
   pum_win_col_offset = 0;
+
+  if (ui_has(kUIMultigrid) && (grid == 0 || grid == 1)) {
+    win_T *wp = mouse_find_win(&grid, &row, &col);
+    // mouse_find_win compensates for the offset due to winbar for example
+    // but we don't want that here
+    row += wp->w_winrow_off;
+    col += wp->w_wincol_off;
+  }
```

Comment:
Is this needed? Won't it be fine to just anchor to grid 0? And it's more accurate, as the window found by `mouse_find_win()` may have been resized by `nvim_ui_try_resize_grid`.

Reply:
Actually never mind.